rockstor.spec

```diff
@@ -370,36 +416,205 @@ exit 0
+-Address compression cosmetics #2640 @StephenBrown2 
+-Update .gitignore with Rockstor build artifacts #2644 @StephenBrown2 
+* Wed Aug 02 2023 The Rockstor Project <support@rockstor.com> - 5.0.2-0
+-resolve indeterminate or inappropriate postgresql alternative conf #2632 @phillxnet 
+-Add option for ZSTD compression #2618 @StephenBrown2 
+-Improve OS independence re unit tests #2633 @phillxnet 
+-user admin enable without password change not internal error #2635 @phillxnet 
+* Mon Jul 24 2023 The Rockstor Project <support@rockstor.com> - 5.0.1-0
+-venv not updating re python version change #2626 @phillxnet 
+-(t) Failure to check for updates in locales other than en_US #2627 @phillxnet 
+* Fri Jul 21 2023 The Rockstor Project <support@rockstor.com> - 5.0.0-0
+-WARNING -- RENEWED TESTING PHASE 5.0.0-0 -- (DEVELOPERS ONLY) #2610 @phillxnet 
+-Update specifics of Python version in pyproject.toml etc #2620 @phillxnet 
+-Update unit tests re recent SSL Cert update changes #2611 @phillxnet 
+-update use of python distro module re 1.7.0+ changes #2426 @phillxnet 
+-Disposition of pytz package #2590 @phillxnet 
+-SSL Certificate update doesn't reload nginx #2606 @phillxnet 
+-Testing counterpart Revise Stable Updates activation re legacy shop removal #2603 @phillxnet 
+-Add GitHub Action to trigger post-release updates across repositories #2600  @FroggyFlox 
+-Restore config backup functionality #2569 @FroggyFlox 
+-Py3.6 Update certifi, urllib3, idna #2588 @phillxnet 
+-Restore Logs Manager - Logs Readers functions #2568 @FroggyFlox 
+-Py3.6 Scheduled tasks import paths fix, snapshot & scub #2570 @phillxnet 
+-Py3.6 test_user.py fail re string interpretation #2582 @phillxnet 
+-Save network usage processed data as List object #2577 @FroggyFlox 
+-Py3.6 test_shares.py & test_snapshot.py re boolean type #2580 @phillxnet 
+-Py3.6 ModuleNotFoundError: 'crontabwindow', 'nfsd_calls' - tests #2578 @phillxnet 
+-Py3.6 ModuleNotFoundError: No module named 'mock' - tests #2575 @phillxnet 
+-Py3.6 update test_btrfs.py and use built-in mock #2571 @phillxnet 
+-Preliminary python 3.6 port - development #2564 @phillxnet
 * Wed Jul 12 2023 The Rockstor Project <support@rockstor.com> - 4.6.1-0
--Bump versions to a 4.6.1 base (Stable) - master branch #2615 @phillxnet 
--Update unit tests re recent SSL Cert update changes #2613 @phillxnet 
--SSL Certificate update doesn't reload nginx #2566 @phillxnet 
--Revise Stable Updates activation re legacy shop removal #2599 @phillxnet 
--Add GitHub Action to trigger post-release updates across repositories #2596 @FroggyFlox 
+-Bump versions to a 4.6.1 base (Stable) - master branch #2615 @phillxnet
+-Update unit tests re recent SSL Cert update changes #2613 @phillxnet
+-SSL Certificate update doesn't reload nginx #2566 @phillxnet
+-Revise Stable Updates activation re legacy shop removal #2599 @phillxnet
+-Add GitHub Action to trigger post-release updates across repositories #2596 @FroggyFlox
```

Comment:
Mostly harmless end-of-line char anomalies here that we have seen before - noting for clarification of this non-visible change.